web/js/validation.js

```diff
@@ -0,0 +1,4 @@
+(function () {
+    var regex = "[a-zA-Z0-9_]+";
+    $("#user_username").attr("pattern", regex);
```

Comment:
wel bizar is dat het in principe al zou moeten gebeuren omwille van de symfony validation ... zal een bug zijn in die validation.  Je kan wel dezelfde regex gebruiken uit de Person Entity ;-)